DatadogLogs/Sources/RemoteLogger.swift

```diff
@@ -189,6 +189,15 @@ internal final class RemoteLogger: LoggerProtocol {
                     return
                 }
 
+                // Add back in fingerprint and error source type
+                var busCombinedAttributes = combinedAttributes
+                if let errorSourcetype = error?.sourceType {
+                    busCombinedAttributes[CrossPlatformAttributes.errorSourceType] = errorSourcetype
+                }
+                if let errorFingerprint = errorFingerprint {
+                    busCombinedAttributes[Logs.Attributes.errorFingerprint] = errorFingerprint
+                }
+
```

Comment:
/suggestion I think it would be clearer if that these properties were added to `ErrorMessage` object, would that be possible?

Reply:
That's a much larger refactor is the issue. 😓